modules/desktop-virtualization/workspace/MOVED-TO-AVM.md

```diff
@@ -0,0 +1 @@
+This module has already been migrated to [AVM](https://github.com/Azure/bicep-registry-modules/tree/main/avm/res). Only the AVM version is expected to receive updates / new features. Please do not work on improving this module in [CARML](https://aka.ms/carml).
```

Comment:
Note, this require and update for the Readme as well. Like in AVM, please run the `utilities/tools/Set-Module.ps1` script on the module folder

Reply:
> Note, this require and update for the Readme as well. Like in AVM, please run the `utilities/tools/Set-Module.ps1` script on the module folder

Readme updated.